### PR TITLE
Update: show access stats error message on 403

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #460

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/SidebarAccessStats.js [hunk 1]
```diff
@@ -29,20 +30,24 @@ const SidebarAccessStats = ({
...
+    intl
 }: Props) => {
     const { preview_count, comment_count, download_count, edit_count } = accessStats;
 
-    if (!preview_count && !comment_count && !download_count && !edit_count) {
+    if (!preview_count && !comment_count && !download_count && !edit_count && !error) {
         return null;
     }
 
+    const errorMessage = error ? intl.formatMessage(error) : undefined;
```
Comment (priyajeet) [inline src/components/ContentSidebar/SidebarAccessStats.js]: Fine for now, but ideally you should go change AccessStates.js to take in a Node in BRUI.

Comment (DanDeMicco) [reply src/components/ContentSidebar/SidebarAccessStats.js]: Agreed, technically it looks like it can, but the flow type says its a string

**Pull request merged**